common/network-defaults/build.rs

```diff
@@ -0,0 +1,64 @@
+use regex::Regex;
+use std::{collections::HashMap, fs};
+
+const TARGET_ENV_FILE: &str = "../../envs/mainnet.env";
```

Comment:
I wonder about this, do you think it's possible to make this relative to the repo root directory? Soon I would like to move the root Rust workspace to a subdirectory to avoid the workspace nesting, and this would then break

Also put in a check that we are writing to the directory we think we are. So that it doesn't try to write the file unless the directory path is correct

Reply:
> make this relative to the repo root directory

done

> Also put in a check that we are writing to the directory we think we are. So that it doesn't try to write the file unless the directory path is correct

done